### ISSUE TITLE
DateTime picker issue on tablet

### ISSUE THREAD

**Issue by Progdrasil** [vanpra/compose-material-dialogs#74]
Hello i'm getting the following output on my 8in tablet. I'm using these in a compose view if that does any difference.

![Screenshot_20210527-171219_timepicker](https://user-images.githubusercontent.com/11562522/119897863-0a1cb400-bf0f-11eb-892d-1d23e65de141.jpg)
![Screenshot_20210527-165639_datepicker](https://user-images.githubusercontent.com/11562522/119897918-186ad000-bf0f-11eb-8332-df56fcde08a7.jpg)

**Comment by Progdrasil** [comment]
The compose view i'm using this in is in a recycler view with the following layout xml:
```xml
<?xml version="1.0" encoding="utf-8"?>
<layout xmlns:android="http://schemas.android.com/apk/res/android">

    <com.google.android.material.card.MaterialCardView
        android:id="@+id/card"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_marginTop="16dp"
        android:layout_marginBottom="16dp">

        <androidx.compose.ui.platform.ComposeView
            android:id="@+id/compose_view"
            android:layout_width="match_parent"
            android:layout_height="match_parent" />
    </com.google.android.material.card.MaterialCardView>
</layout>
```

**Comment by vanpra** [comment]
Thanks for letting me know. I'll take a look into this but for now I'll probably stretch the layout to fix this but in the future I'll look into making specialised layouts to utilise the extra space. Will try to get this fixed for the next compose release coming this week.

**Comment by vanpra** [comment]
Should be fixed in the 0.4.2 release
